enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -622,58 +622,178 @@ will be reconciled in any namespaces, removing the need for ServiceMeshMemberRol
 Cluster-scoped mode is a Tech Preview feature in OSSM 2.3 and fully supported in
 OSSM 2.4.
 
-#### Security Policy
+#### Gateway Topology
+
+Users have the option to deploy their Gateways using two distinct topologies: shared gateways or dedicated gateways.
+Each topology addresses specific customer requirements, comes with different security implications (see [Security Policy](#security-policy)),
+and will be supported in OpenShift.
+
+##### Shared Gateway Topology
+
+With shared gateway topology, a single load balancer type service and proxy deployment is created and serves
+routes across multiple namespaces. The Gateway must allow the desired application namespaces via the
+`spec.listeners.allowedRoutes.namespaces` field. This topology mirrors the existing OpenShift router topology and
+is ideal for users that are cost sensitive and do not have strict performance or security requirements.
+
+```mermaid
+flowchart TD
+    subgraph foo-app namespace
+        xRoute
+    end
+    subgraph bar-app namespace
+        xRoute2
+    end
+    subgraph openshift-ingress namespace
+        Gateway
+        ProxyDeployment
+        LBService
+    end
+    Gateway[Gateway] -.-> ProxyDeployment(Proxy Deployment)
+    Gateway[Gateway] -.-> LBService(LB Service)
```

Comment:
This diagram is also a little murky, as it breaks the abstraction at the Proxy Deployment and LB Service level. These two objects can be replaced by Service, as the Deployment is not a user-managed object. There can also be multiple Services.
```suggestion
    Gateway[Gateway] -.-> Service(Service)
```